src/components/VideoPlayerContexts/PlaybackContext.tsx

```diff
@@ -69,12 +60,9 @@ function PlaybackContextProvider({children}) {
     );
 
     const checkVideoPlaying = useCallback(
-        (statusCallback) => {
-            if (!(currentVideoPlayerRef && currentVideoPlayerRef.current && currentVideoPlayerRef.current.getStatusAsync)) {
-                return;
-            }
-            currentVideoPlayerRef.current.getStatusAsync().then((status) => {
-                statusCallback(status.isPlaying);
+        (statusCallback: StatusCallback) => {
+            currentVideoPlayerRef.current?.getStatusAsync().then((status) => {
+                statusCallback('isPlaying' in status && status.isPlaying);
```

Comment:
why did you add `'isPlaying' in status` here? 😄

Reply:
can we do `status instanceof AVPlaybackStatusSuccess`?

Reply:
`AVPlaybackStatusSuccess` is not a class so I don't think we can :/